examples/3d_examples/MagneticCiliaCarpetCase/magnetic_cilia_carpet.py

```diff
@@ -7,12 +7,18 @@
 class MagneticCiliaCarpetSimulator:
     def __init__(
         self,
+        magnetic_elastic_ratio: float,  # MBAL2_EI, (Wang 2019)
```

Comment:
Lets call this magnetic bond number and maybe put a comment saying what it is.

Reply:
[ce91966](https://github.com/SophT-Team/SophT/pull/120/commits/ce919664854a349d4b924b8ee527419355988c09)